pkg/utils/vdpa_provider.go

```diff
@@ -25,5 +27,14 @@ func GetVdpaProvider() VdpaProvider {
 }
 
 func (defaultVdpaProvider) GetVdpaDeviceByPci(pciAddr string) (vdpa.VdpaDevice, error) {
-	return vdpa.GetVdpaDeviceByPci(pciAddr)
+	// the govdpa library requires the pci address to include the "pci/" prefix
+	fullPciAddr := "pci/" + pciAddr
+	vdpaDevices, err := vdpa.GetVdpaDevicesByPciAddress(fullPciAddr)
+	if err != nil {
+		return nil, err
+	}
+	if len(vdpaDevices) == 0 {
+		return nil, fmt.Errorf("no vdpa device associated to pciAddress %v", pciAddr)
+	}
+	return vdpaDevices[0], nil
```

Comment:
so there may be multiple VDPA devices per PCI addr ? and we currently just care of the first one ?

Reply:
Maybe add a logging message that we are ignoring all but the first